src/NzbDrone.Common/Disk/DriveInfoMount.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using NzbDrone.Common.Extensions;
+
+namespace NzbDrone.Common.Disk
+{
+    public class DriveInfoMount : IMount
```

Comment:
No it's a wrapper around DriveInfo, implementing IMount. Like ProcMount wraps /proc.